### ISSUE TITLE
Docker? 

### ISSUE THREAD

**Issue by droopified** [danxfisher/Plex-Discord-Bot#17]
Any chance your can make a docker for this?

**Comment by twilsonco** [comment]
There's a docker [here](https://github.com/MoSadie/Plex-Discord-Bot) (I haven't tested it)

**Comment by irbyk** [comment]
I've done a docker file [here](https://github.com/irbyk/Plex-Discord-Bot_V2), you can use it with this bot (be carefull with the depences. The package.json in the danxfisher repository is out of date ...).

**Comment by JoshuaMunoz** [comment]
Is a docker like an actual dock where you can more easily select your music instead of having to punch in the file name or something? More visual, GUI-type way of running the bot?

**Comment by maverickmarq** [comment]
> Is a docker like an actual dock where you can more easily select your music instead of having to punch in the file name or something? More visual, GUI-type way of running the bot?

No. It's a way of deploying an environment on a server, basically. Very much not more visual.
